src/commands/sfpowerkit/project/datamodel/diff.ts

```diff
@@ -0,0 +1,205 @@
+      "--short",
+      this.flags.revisionto
+    ]);
+
+    let packageDirectories: string[];
+
+    if (!isNullOrUndefined(this.flags.packagedirectories)) {
+      packageDirectories = this.flags.packagedirectories.split(",");
+      packageDirectories = packageDirectories.map(dir => {
+        return dir.trim().toLocaleLowerCase();
+      });
+
+      let projectConfig = JSON.parse(
+        fs.readFileSync("sfdx-project.json", "utf8")
+      );
+      packageDirectories.forEach(dir => {
+        let isValidPackageDir: boolean;
+        projectConfig["packageDirectories"].forEach(configPackageDir => {
+          if (dir == configPackageDir["path"].toLocaleLowerCase())
+            isValidPackageDir = true;
+        });
+        if (!isValidPackageDir)
+          throw new Error("Invalid package directory supplied");
+      });
+    }
+
+    let dataModelSourceDiffImpl = new DataModelSourceDiffImpl(
+      git,
+      revisionFrom,
+      revisionTo,
+      packageDirectories
+    );
+
+    let sourceDiffResult = await dataModelSourceDiffImpl.exec();
+
+    if (sourceDiffResult.length < 1) {
+      SFPowerkit.log(
+        `No Datamodel change found between ${revisionFrom} and ${revisionTo}`,
+        LoggerLevel.WARN
+      );
+      return sourceDiffResult;
+    }
+
+    SFPowerkit.log(
+      `Found ${sourceDiffResult.length} Datamodel change between ${revisionFrom} and ${revisionTo} \n`,
+      LoggerLevel.INFO
+    );
+
+    let csvPath = `${outputDirectory}/datamodel-diff-output.json`;
+    let dir = path.parse(csvPath).dir;
+    if (!fs.existsSync(dir)) {
+      FileUtils.mkDirByPathSync(dir);
+    }
+    fs.writeFileSync(csvPath, JSON.stringify(sourceDiffResult, null, 4));
+
+    let rowsToDisplay = [];
+    for (let file of sourceDiffResult) {
+      for (let change of file["diff"]) {
+        rowsToDisplay.push({
+          object: file["object"],
+          api_name: file["api_name"],
+          type: file["type"],
+          operation: change["operation"],
+          coordinates: change["coordinates"],
+          from: change["before"],
+          to: change["after"],
+          filepath: file["filepath"]
+        });
+      }
+    }
+
+    if (isOutputCSV) {
+      let csvOutput: string = `Object,API_Name,Type,Operation,Coordinates,Commit ID (${revisionFrom}),Commit ID (${revisionTo}),Filepath\n`;
+
+      for (let row of rowsToDisplay) {
+        let rowCells: string[] = Object.values(row);
+        csvOutput = csvOutput + `${rowCells.toString()}\n`;
+      }
+
+      fs.writeFileSync(
+        `${outputDirectory}/datamodel-diff-output.csv`,
+        csvOutput
+      );
+    }
+
+    this.ux.table(rowsToDisplay.slice(0, 50), [
+      "object",
+      "api_name",
+      "type",
+      "operation",
+      "coordinates",
+      "from",
+      "to"
+    ]);
+    this.ux.log("\n");
+    if (rowsToDisplay.length > 50) {
+      SFPowerkit.log("Displaying output limited to 50 rows", LoggerLevel.WARN);
```

Comment:
```suggestion
      SFPowerkit.log("Displaying output limited to 50 rows", LoggerLevel.INFO);
```
Info should be enough

Reply:
I prefer warning as it shows in a different colour, prompting the user to go check the file for the full results.

Reply:
Warn should be fine